### ISSUE TITLE
Create a MLE model to predict match outcomes 

### ISSUE THREAD

**Issue by tkpratardan** [kaizen-ai/kaizenflow#1029]
Use the MLE estimator to predict match outcomes on the probability created in sota_baseline_glm.ipynb.

**Comment by gpsaggese** [comment]
Epic and estimate pls.

So you want to estimate the raw probabilities of the classes from the train set and use them in the test set, right?

Not sure what's the simplest way to do this with sklearn. I'd spend 10-15 mins going through the docs to see if there is a simple way to do it, otherwise you can just derive from the base model
https://scikit-learn.org/stable/developers/develop.html

If you follow this approach then it's super simple to do a comparison btw models, since you just need to pass the model around and the entire code base is the same.

We should go towards this approach in any case, including for the Poisson regression.

Our approach is to always implement the sklearn interfaces, so they also plug in KaizenFlow right away.

Your call if you want to do a quick prototype or do the right approach from the beginning.

**Comment by sonaalKant** [comment]
Closing, this is done and merged. https://github.com/kaizen-ai/kaizenflow/pull/1049
